filecoin-proofs/src/api/seal.rs

```diff
@@ -299,7 +301,8 @@ where
 
     // Persist p_aux and t_aux here
     util::persist_p_aux::<Tree>(&p_aux, cache_path.as_ref())?;
-    util::persist_t_aux::<Tree>(&t_aux, cache_path.as_ref())?;
+    #[cfg(not(feature = "fixed-rows-to-discard"))]
```

Comment:
Last nit on this -- could we instead have a `persist_t_aux` defined for this feature config that is more or less an inline'd no-op?  I think the switching at this level is still not optimal because it's not clear that `fixed-rows-to-discard` means we don't need `t_aux`, although this is an initial case where that's true.  It also consolidates all switching on this config related to the p_aux/t_aux to the util methods.

Reply:
> I think the switching at this level is still not optimal because it's not clear that `fixed-rows-to-discard` means we don't need `t_au`

I think of it as the reverse. If I would read the code and see a `persist_t_aux` function call, i would think "oh, t_aux is always persisted". Only if I then look at the implementation I see that it's actually not persisted in some cases. In the current code I see it right away that if that feature is enabled, nothing will be persisted. I would even see that there isn't even a persist_t_aux implementation at all.